open-company/open-company-web
# Fix attachments layout and delete button.

Abstract: https://share.goabstract.com/55222102-06e9-4a50-9832-dcb10b7c7e4c

To test:
- add an attachment
- remove it
- check that the attachments style correspond to abstract design
